bpython/curtsiesfrontend/repl.py

```diff
@@ -844,6 +846,13 @@ def on_control_d(self):
             self.current_line = (self.current_line[:self.cursor_offset] +
                                  self.current_line[(self.cursor_offset + 1):])
 
+    def on_control_o(self):
```

Comment:
Actually maybe the same escape hatch should apply to `self.rl_history.index == 1` as well, because this math doesn't work in that case either (the case where we're looking at the last entry in history).